sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -26,16 +26,17 @@ export interface BeginRecoverDeletedKeyOptions extends KeyPollerOptions {
 
 // @public
 export interface CreateEcKeyOptions extends CreateKeyOptions {
-    curve?: KeyCurveName;
-    hsm?: boolean;
+    curve?: KeyCurveName; // Should we rename to curveName and deprecate curve?
```

Comment:
If you GA'd this way, I wouldn't worry about changing. Seems it would cause more harm than good.